stable_baselines3/common/off_policy_algorithm.py

```diff
@@ -83,9 +81,8 @@ def __init__(
         batch_size: int = 256,
         tau: float = 0.005,
         gamma: float = 0.99,
-        train_freq: int = 1,
+        train_freq: Union[int, Tuple[int, str]] = (1, "episode"),
```

Comment:
please keep the default ("step")

Reply:
I was assuming the default should be 1 step? So either `1` or `(1, "step")` should be fine? Why are you suggesting 1 episode? Or am I getting something wrong here?

Reply:
yes, the default should be one step (but that's not what it is right now, no?) I read `(1, "episode"),`

Reply:
Fixed it